include/solvers/timesteppers/FoamControlledTimeStepper.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "FoamProblem.h"
+#include "FoamSolver.h"
+#include "InputParameters.h"
+
```

Comment:
This is a MOOSE include so should go in the include block below and use `<>` brackets.

I should have documented the include conventions for the repo (my bad). But Hippo includes are first (using `""`). Then third party includes (so MOOSE/OpenFOAM, using `<>`), then standard library includes (using `<>`). Not massively important, but it keeps things organised and easy to parse by eye.

```suggestion

#include <InputParameters.h>
```